frontend/src/components/BottomNav.tsx

```diff
@@ -9,7 +9,7 @@ export default function BottomNav() {
     const [value, setValue] = React.useState(location.pathname);
 
     return (
-        <Paper sx={{position: 'fixed', bottom: 0, left: 0, right: 0}} elevation={3}>
+        <Paper sx={{position: 'fixed', bottom: 0, left: 0, right: 0, zIndex: (theme) => theme.zIndex.appBar - 1}} elevation={10}>
```

Comment:
Is that a thing for MUI components to use inline style? Or rather your personal preference?

Reply:
Jap, that‘s inline styling for MUI components. You could use styled components as a cleaner approach, but I haven‘t dived into it yet. So I‘ll stick with using the sx property for now